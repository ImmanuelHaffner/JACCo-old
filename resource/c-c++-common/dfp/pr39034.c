





int failures;



void
runtime32 (void)
{
  volatile float v1 = 28.f, v2 = 3.f, v3 = 9.f, v4 = 31.f, v5 = 3.f, v6 = 10.f;
  float b32 = (float)((v1/v2-v3) - (v4/v5-v6));
  _Decimal32 d32 = (float)((v1/v2-v3) - (v4/v5-v6));

  if (b32)
    __builtin_abort ();
  if (d32)
    __builtin_abort ();
}

void
runtime64 (void)
{
  volatile double v1 = 28., v2 = 3., v3 = 9., v4 = 31., v5 = 3., v6 = 10.;
  double b64 = (double)((v1/v2-v3) - (v4/v5-v6));
  _Decimal64 d64 = (double)((v1/v2-v3) - (v4/v5-v6));

  if (b64)
    __builtin_abort ();
  if (d64)
    __builtin_abort ();
}

void
runtime128 (void)
{
  volatile long double v1 = 28.l, v2 = 3.l, v3 = 9.l,
                       v4 = 31.l, v5 = 3.l, v6 = 10.l;
  long double b128 = (long double)((v1/v2-v3) - (v4/v5-v6));
  _Decimal128 d128 = (long double)((v1/v2-v3) - (v4/v5-v6));

  if (b128)
    __builtin_abort ();
  if (d128)
    __builtin_abort ();
}



void
fold32 (void)
{
  double d32 = (float)((28.f/3.f-9.f) - (31.f/3.f-10.f));
  _Decimal32 b32 = (float)((28.f/3.f-9.f) - (31.f/3.f-10.f));

  if (b32)
    __builtin_abort ();
  if (d32)
    __builtin_abort ();
}

void
fold64 (void)
{
  double b64 = (double)((28./3.-9.) - (31./3.-10.));
  _Decimal64 d64 = (double)((28./3.-9.) - (31./3.-10.));

  if (b64)
    __builtin_abort ();
  if (d64)
    __builtin_abort ();
}

void
fold128 (void)
{
  long double b128 = (long double)((28./3.-9.) - (31./3.-10.));
  _Decimal128 d128 = (long double)((28./3.-9.) - (31./3.-10.));

  if (b128)
    __builtin_abort ();
  if (d128)
    __builtin_abort ();
}

int
main ()
{
  runtime32 ();
  runtime64 ();
  runtime128 ();
  fold32 ();
  fold64 ();
  fold128 ();

  return 0;
}
