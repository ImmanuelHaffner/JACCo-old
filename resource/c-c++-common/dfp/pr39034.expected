0
pr39034.c:7:1 keyword int
pr39034.c:7:5 identifier failures
pr39034.c:7:13 punctuator ;
pr39034.c:11:1 keyword void
pr39034.c:12:1 identifier runtime32
pr39034.c:12:11 punctuator (
pr39034.c:12:12 keyword void
pr39034.c:12:16 punctuator )
pr39034.c:13:1 punctuator {
pr39034.c:14:3 keyword volatile
pr39034.c:14:12 keyword float
pr39034.c:14:18 identifier v1
pr39034.c:14:21 punctuator =
pr39034.c:14:23 constant 28.f
pr39034.c:14:27 punctuator ,
pr39034.c:14:29 identifier v2
pr39034.c:14:32 punctuator =
pr39034.c:14:34 constant 3.f
pr39034.c:14:37 punctuator ,
pr39034.c:14:39 identifier v3
pr39034.c:14:42 punctuator =
pr39034.c:14:44 constant 9.f
pr39034.c:14:47 punctuator ,
pr39034.c:14:49 identifier v4
pr39034.c:14:52 punctuator =
pr39034.c:14:54 constant 31.f
pr39034.c:14:58 punctuator ,
pr39034.c:14:60 identifier v5
pr39034.c:14:63 punctuator =
pr39034.c:14:65 constant 3.f
pr39034.c:14:68 punctuator ,
pr39034.c:14:70 identifier v6
pr39034.c:14:73 punctuator =
pr39034.c:14:75 constant 10.f
pr39034.c:14:79 punctuator ;
pr39034.c:15:3 keyword float
pr39034.c:15:9 identifier b32
pr39034.c:15:13 punctuator =
pr39034.c:15:15 punctuator (
pr39034.c:15:16 keyword float
pr39034.c:15:21 punctuator )
pr39034.c:15:22 punctuator (
pr39034.c:15:23 punctuator (
pr39034.c:15:24 identifier v1
pr39034.c:15:26 punctuator /
pr39034.c:15:27 identifier v2
pr39034.c:15:29 punctuator -
pr39034.c:15:30 identifier v3
pr39034.c:15:32 punctuator )
pr39034.c:15:34 punctuator -
pr39034.c:15:36 punctuator (
pr39034.c:15:37 identifier v4
pr39034.c:15:39 punctuator /
pr39034.c:15:40 identifier v5
pr39034.c:15:42 punctuator -
pr39034.c:15:43 identifier v6
pr39034.c:15:45 punctuator )
pr39034.c:15:46 punctuator )
pr39034.c:15:47 punctuator ;
pr39034.c:16:3 identifier _Decimal32
pr39034.c:16:14 identifier d32
pr39034.c:16:18 punctuator =
pr39034.c:16:20 punctuator (
pr39034.c:16:21 keyword float
pr39034.c:16:26 punctuator )
pr39034.c:16:27 punctuator (
pr39034.c:16:28 punctuator (
pr39034.c:16:29 identifier v1
pr39034.c:16:31 punctuator /
pr39034.c:16:32 identifier v2
pr39034.c:16:34 punctuator -
pr39034.c:16:35 identifier v3
pr39034.c:16:37 punctuator )
pr39034.c:16:39 punctuator -
pr39034.c:16:41 punctuator (
pr39034.c:16:42 identifier v4
pr39034.c:16:44 punctuator /
pr39034.c:16:45 identifier v5
pr39034.c:16:47 punctuator -
pr39034.c:16:48 identifier v6
pr39034.c:16:50 punctuator )
pr39034.c:16:51 punctuator )
pr39034.c:16:52 punctuator ;
pr39034.c:18:3 keyword if
pr39034.c:18:6 punctuator (
pr39034.c:18:7 identifier b32
pr39034.c:18:10 punctuator )
pr39034.c:19:5 identifier __builtin_abort
pr39034.c:19:21 punctuator (
pr39034.c:19:22 punctuator )
pr39034.c:19:23 punctuator ;
pr39034.c:20:3 keyword if
pr39034.c:20:6 punctuator (
pr39034.c:20:7 identifier d32
pr39034.c:20:10 punctuator )
pr39034.c:21:5 identifier __builtin_abort
pr39034.c:21:21 punctuator (
pr39034.c:21:22 punctuator )
pr39034.c:21:23 punctuator ;
pr39034.c:22:1 punctuator }
pr39034.c:24:1 keyword void
pr39034.c:25:1 identifier runtime64
pr39034.c:25:11 punctuator (
pr39034.c:25:12 keyword void
pr39034.c:25:16 punctuator )
pr39034.c:26:1 punctuator {
pr39034.c:27:3 keyword volatile
pr39034.c:27:12 keyword double
pr39034.c:27:19 identifier v1
pr39034.c:27:22 punctuator =
pr39034.c:27:24 constant 28.
pr39034.c:27:27 punctuator ,
pr39034.c:27:29 identifier v2
pr39034.c:27:32 punctuator =
pr39034.c:27:34 constant 3.
pr39034.c:27:36 punctuator ,
pr39034.c:27:38 identifier v3
pr39034.c:27:41 punctuator =
pr39034.c:27:43 constant 9.
pr39034.c:27:45 punctuator ,
pr39034.c:27:47 identifier v4
pr39034.c:27:50 punctuator =
pr39034.c:27:52 constant 31.
pr39034.c:27:55 punctuator ,
pr39034.c:27:57 identifier v5
pr39034.c:27:60 punctuator =
pr39034.c:27:62 constant 3.
pr39034.c:27:64 punctuator ,
pr39034.c:27:66 identifier v6
pr39034.c:27:69 punctuator =
pr39034.c:27:71 constant 10.
pr39034.c:27:74 punctuator ;
pr39034.c:28:3 keyword double
pr39034.c:28:10 identifier b64
pr39034.c:28:14 punctuator =
pr39034.c:28:16 punctuator (
pr39034.c:28:17 keyword double
pr39034.c:28:23 punctuator )
pr39034.c:28:24 punctuator (
pr39034.c:28:25 punctuator (
pr39034.c:28:26 identifier v1
pr39034.c:28:28 punctuator /
pr39034.c:28:29 identifier v2
pr39034.c:28:31 punctuator -
pr39034.c:28:32 identifier v3
pr39034.c:28:34 punctuator )
pr39034.c:28:36 punctuator -
pr39034.c:28:38 punctuator (
pr39034.c:28:39 identifier v4
pr39034.c:28:41 punctuator /
pr39034.c:28:42 identifier v5
pr39034.c:28:44 punctuator -
pr39034.c:28:45 identifier v6
pr39034.c:28:47 punctuator )
pr39034.c:28:48 punctuator )
pr39034.c:28:49 punctuator ;
pr39034.c:29:3 identifier _Decimal64
pr39034.c:29:14 identifier d64
pr39034.c:29:18 punctuator =
pr39034.c:29:20 punctuator (
pr39034.c:29:21 keyword double
pr39034.c:29:27 punctuator )
pr39034.c:29:28 punctuator (
pr39034.c:29:29 punctuator (
pr39034.c:29:30 identifier v1
pr39034.c:29:32 punctuator /
pr39034.c:29:33 identifier v2
pr39034.c:29:35 punctuator -
pr39034.c:29:36 identifier v3
pr39034.c:29:38 punctuator )
pr39034.c:29:40 punctuator -
pr39034.c:29:42 punctuator (
pr39034.c:29:43 identifier v4
pr39034.c:29:45 punctuator /
pr39034.c:29:46 identifier v5
pr39034.c:29:48 punctuator -
pr39034.c:29:49 identifier v6
pr39034.c:29:51 punctuator )
pr39034.c:29:52 punctuator )
pr39034.c:29:53 punctuator ;
pr39034.c:31:3 keyword if
pr39034.c:31:6 punctuator (
pr39034.c:31:7 identifier b64
pr39034.c:31:10 punctuator )
pr39034.c:32:5 identifier __builtin_abort
pr39034.c:32:21 punctuator (
pr39034.c:32:22 punctuator )
pr39034.c:32:23 punctuator ;
pr39034.c:33:3 keyword if
pr39034.c:33:6 punctuator (
pr39034.c:33:7 identifier d64
pr39034.c:33:10 punctuator )
pr39034.c:34:5 identifier __builtin_abort
pr39034.c:34:21 punctuator (
pr39034.c:34:22 punctuator )
pr39034.c:34:23 punctuator ;
pr39034.c:35:1 punctuator }
pr39034.c:37:1 keyword void
pr39034.c:38:1 identifier runtime128
pr39034.c:38:12 punctuator (
pr39034.c:38:13 keyword void
pr39034.c:38:17 punctuator )
pr39034.c:39:1 punctuator {
pr39034.c:40:3 keyword volatile
pr39034.c:40:12 keyword long
pr39034.c:40:17 keyword double
pr39034.c:40:24 identifier v1
pr39034.c:40:27 punctuator =
pr39034.c:40:29 constant 28.l
pr39034.c:40:33 punctuator ,
pr39034.c:40:35 identifier v2
pr39034.c:40:38 punctuator =
pr39034.c:40:40 constant 3.l
pr39034.c:40:43 punctuator ,
pr39034.c:40:45 identifier v3
pr39034.c:40:48 punctuator =
pr39034.c:40:50 constant 9.l
pr39034.c:40:53 punctuator ,
pr39034.c:41:24 identifier v4
pr39034.c:41:27 punctuator =
pr39034.c:41:29 constant 31.l
pr39034.c:41:33 punctuator ,
pr39034.c:41:35 identifier v5
pr39034.c:41:38 punctuator =
pr39034.c:41:40 constant 3.l
pr39034.c:41:43 punctuator ,
pr39034.c:41:45 identifier v6
pr39034.c:41:48 punctuator =
pr39034.c:41:50 constant 10.l
pr39034.c:41:54 punctuator ;
pr39034.c:42:3 keyword long
pr39034.c:42:8 keyword double
pr39034.c:42:15 identifier b128
pr39034.c:42:20 punctuator =
pr39034.c:42:22 punctuator (
pr39034.c:42:23 keyword long
pr39034.c:42:28 keyword double
pr39034.c:42:34 punctuator )
pr39034.c:42:35 punctuator (
pr39034.c:42:36 punctuator (
pr39034.c:42:37 identifier v1
pr39034.c:42:39 punctuator /
pr39034.c:42:40 identifier v2
pr39034.c:42:42 punctuator -
pr39034.c:42:43 identifier v3
pr39034.c:42:45 punctuator )
pr39034.c:42:47 punctuator -
pr39034.c:42:49 punctuator (
pr39034.c:42:50 identifier v4
pr39034.c:42:52 punctuator /
pr39034.c:42:53 identifier v5
pr39034.c:42:55 punctuator -
pr39034.c:42:56 identifier v6
pr39034.c:42:58 punctuator )
pr39034.c:42:59 punctuator )
pr39034.c:42:60 punctuator ;
pr39034.c:43:3 identifier _Decimal128
pr39034.c:43:15 identifier d128
pr39034.c:43:20 punctuator =
pr39034.c:43:22 punctuator (
pr39034.c:43:23 keyword long
pr39034.c:43:28 keyword double
pr39034.c:43:34 punctuator )
pr39034.c:43:35 punctuator (
pr39034.c:43:36 punctuator (
pr39034.c:43:37 identifier v1
pr39034.c:43:39 punctuator /
pr39034.c:43:40 identifier v2
pr39034.c:43:42 punctuator -
pr39034.c:43:43 identifier v3
pr39034.c:43:45 punctuator )
pr39034.c:43:47 punctuator -
pr39034.c:43:49 punctuator (
pr39034.c:43:50 identifier v4
pr39034.c:43:52 punctuator /
pr39034.c:43:53 identifier v5
pr39034.c:43:55 punctuator -
pr39034.c:43:56 identifier v6
pr39034.c:43:58 punctuator )
pr39034.c:43:59 punctuator )
pr39034.c:43:60 punctuator ;
pr39034.c:45:3 keyword if
pr39034.c:45:6 punctuator (
pr39034.c:45:7 identifier b128
pr39034.c:45:11 punctuator )
pr39034.c:46:5 identifier __builtin_abort
pr39034.c:46:21 punctuator (
pr39034.c:46:22 punctuator )
pr39034.c:46:23 punctuator ;
pr39034.c:47:3 keyword if
pr39034.c:47:6 punctuator (
pr39034.c:47:7 identifier d128
pr39034.c:47:11 punctuator )
pr39034.c:48:5 identifier __builtin_abort
pr39034.c:48:21 punctuator (
pr39034.c:48:22 punctuator )
pr39034.c:48:23 punctuator ;
pr39034.c:49:1 punctuator }
pr39034.c:53:1 keyword void
pr39034.c:54:1 identifier fold32
pr39034.c:54:8 punctuator (
pr39034.c:54:9 keyword void
pr39034.c:54:13 punctuator )
pr39034.c:55:1 punctuator {
pr39034.c:56:3 keyword double
pr39034.c:56:10 identifier d32
pr39034.c:56:14 punctuator =
pr39034.c:56:16 punctuator (
pr39034.c:56:17 keyword float
pr39034.c:56:22 punctuator )
pr39034.c:56:23 punctuator (
pr39034.c:56:24 punctuator (
pr39034.c:56:25 constant 28.f
pr39034.c:56:29 punctuator /
pr39034.c:56:30 constant 3.f
pr39034.c:56:33 punctuator -
pr39034.c:56:34 constant 9.f
pr39034.c:56:37 punctuator )
pr39034.c:56:39 punctuator -
pr39034.c:56:41 punctuator (
pr39034.c:56:42 constant 31.f
pr39034.c:56:46 punctuator /
pr39034.c:56:47 constant 3.f
pr39034.c:56:50 punctuator -
pr39034.c:56:51 constant 10.f
pr39034.c:56:55 punctuator )
pr39034.c:56:56 punctuator )
pr39034.c:56:57 punctuator ;
pr39034.c:57:3 identifier _Decimal32
pr39034.c:57:14 identifier b32
pr39034.c:57:18 punctuator =
pr39034.c:57:20 punctuator (
pr39034.c:57:21 keyword float
pr39034.c:57:26 punctuator )
pr39034.c:57:27 punctuator (
pr39034.c:57:28 punctuator (
pr39034.c:57:29 constant 28.f
pr39034.c:57:33 punctuator /
pr39034.c:57:34 constant 3.f
pr39034.c:57:37 punctuator -
pr39034.c:57:38 constant 9.f
pr39034.c:57:41 punctuator )
pr39034.c:57:43 punctuator -
pr39034.c:57:45 punctuator (
pr39034.c:57:46 constant 31.f
pr39034.c:57:50 punctuator /
pr39034.c:57:51 constant 3.f
pr39034.c:57:54 punctuator -
pr39034.c:57:55 constant 10.f
pr39034.c:57:59 punctuator )
pr39034.c:57:60 punctuator )
pr39034.c:57:61 punctuator ;
pr39034.c:59:3 keyword if
pr39034.c:59:6 punctuator (
pr39034.c:59:7 identifier b32
pr39034.c:59:10 punctuator )
pr39034.c:60:5 identifier __builtin_abort
pr39034.c:60:21 punctuator (
pr39034.c:60:22 punctuator )
pr39034.c:60:23 punctuator ;
pr39034.c:61:3 keyword if
pr39034.c:61:6 punctuator (
pr39034.c:61:7 identifier d32
pr39034.c:61:10 punctuator )
pr39034.c:62:5 identifier __builtin_abort
pr39034.c:62:21 punctuator (
pr39034.c:62:22 punctuator )
pr39034.c:62:23 punctuator ;
pr39034.c:63:1 punctuator }
pr39034.c:65:1 keyword void
pr39034.c:66:1 identifier fold64
pr39034.c:66:8 punctuator (
pr39034.c:66:9 keyword void
pr39034.c:66:13 punctuator )
pr39034.c:67:1 punctuator {
pr39034.c:68:3 keyword double
pr39034.c:68:10 identifier b64
pr39034.c:68:14 punctuator =
pr39034.c:68:16 punctuator (
pr39034.c:68:17 keyword double
pr39034.c:68:23 punctuator )
pr39034.c:68:24 punctuator (
pr39034.c:68:25 punctuator (
pr39034.c:68:26 constant 28.
pr39034.c:68:29 punctuator /
pr39034.c:68:30 constant 3.
pr39034.c:68:32 punctuator -
pr39034.c:68:33 constant 9.
pr39034.c:68:35 punctuator )
pr39034.c:68:37 punctuator -
pr39034.c:68:39 punctuator (
pr39034.c:68:40 constant 31.
pr39034.c:68:43 punctuator /
pr39034.c:68:44 constant 3.
pr39034.c:68:46 punctuator -
pr39034.c:68:47 constant 10.
pr39034.c:68:50 punctuator )
pr39034.c:68:51 punctuator )
pr39034.c:68:52 punctuator ;
pr39034.c:69:3 identifier _Decimal64
pr39034.c:69:14 identifier d64
pr39034.c:69:18 punctuator =
pr39034.c:69:20 punctuator (
pr39034.c:69:21 keyword double
pr39034.c:69:27 punctuator )
pr39034.c:69:28 punctuator (
pr39034.c:69:29 punctuator (
pr39034.c:69:30 constant 28.
pr39034.c:69:33 punctuator /
pr39034.c:69:34 constant 3.
pr39034.c:69:36 punctuator -
pr39034.c:69:37 constant 9.
pr39034.c:69:39 punctuator )
pr39034.c:69:41 punctuator -
pr39034.c:69:43 punctuator (
pr39034.c:69:44 constant 31.
pr39034.c:69:47 punctuator /
pr39034.c:69:48 constant 3.
pr39034.c:69:50 punctuator -
pr39034.c:69:51 constant 10.
pr39034.c:69:54 punctuator )
pr39034.c:69:55 punctuator )
pr39034.c:69:56 punctuator ;
pr39034.c:71:3 keyword if
pr39034.c:71:6 punctuator (
pr39034.c:71:7 identifier b64
pr39034.c:71:10 punctuator )
pr39034.c:72:5 identifier __builtin_abort
pr39034.c:72:21 punctuator (
pr39034.c:72:22 punctuator )
pr39034.c:72:23 punctuator ;
pr39034.c:73:3 keyword if
pr39034.c:73:6 punctuator (
pr39034.c:73:7 identifier d64
pr39034.c:73:10 punctuator )
pr39034.c:74:5 identifier __builtin_abort
pr39034.c:74:21 punctuator (
pr39034.c:74:22 punctuator )
pr39034.c:74:23 punctuator ;
pr39034.c:75:1 punctuator }
pr39034.c:77:1 keyword void
pr39034.c:78:1 identifier fold128
pr39034.c:78:9 punctuator (
pr39034.c:78:10 keyword void
pr39034.c:78:14 punctuator )
pr39034.c:79:1 punctuator {
pr39034.c:80:3 keyword long
pr39034.c:80:8 keyword double
pr39034.c:80:15 identifier b128
pr39034.c:80:20 punctuator =
pr39034.c:80:22 punctuator (
pr39034.c:80:23 keyword long
pr39034.c:80:28 keyword double
pr39034.c:80:34 punctuator )
pr39034.c:80:35 punctuator (
pr39034.c:80:36 punctuator (
pr39034.c:80:37 constant 28.
pr39034.c:80:40 punctuator /
pr39034.c:80:41 constant 3.
pr39034.c:80:43 punctuator -
pr39034.c:80:44 constant 9.
pr39034.c:80:46 punctuator )
pr39034.c:80:48 punctuator -
pr39034.c:80:50 punctuator (
pr39034.c:80:51 constant 31.
pr39034.c:80:54 punctuator /
pr39034.c:80:55 constant 3.
pr39034.c:80:57 punctuator -
pr39034.c:80:58 constant 10.
pr39034.c:80:61 punctuator )
pr39034.c:80:62 punctuator )
pr39034.c:80:63 punctuator ;
pr39034.c:81:3 identifier _Decimal128
pr39034.c:81:15 identifier d128
pr39034.c:81:20 punctuator =
pr39034.c:81:22 punctuator (
pr39034.c:81:23 keyword long
pr39034.c:81:28 keyword double
pr39034.c:81:34 punctuator )
pr39034.c:81:35 punctuator (
pr39034.c:81:36 punctuator (
pr39034.c:81:37 constant 28.
pr39034.c:81:40 punctuator /
pr39034.c:81:41 constant 3.
pr39034.c:81:43 punctuator -
pr39034.c:81:44 constant 9.
pr39034.c:81:46 punctuator )
pr39034.c:81:48 punctuator -
pr39034.c:81:50 punctuator (
pr39034.c:81:51 constant 31.
pr39034.c:81:54 punctuator /
pr39034.c:81:55 constant 3.
pr39034.c:81:57 punctuator -
pr39034.c:81:58 constant 10.
pr39034.c:81:61 punctuator )
pr39034.c:81:62 punctuator )
pr39034.c:81:63 punctuator ;
pr39034.c:83:3 keyword if
pr39034.c:83:6 punctuator (
pr39034.c:83:7 identifier b128
pr39034.c:83:11 punctuator )
pr39034.c:84:5 identifier __builtin_abort
pr39034.c:84:21 punctuator (
pr39034.c:84:22 punctuator )
pr39034.c:84:23 punctuator ;
pr39034.c:85:3 keyword if
pr39034.c:85:6 punctuator (
pr39034.c:85:7 identifier d128
pr39034.c:85:11 punctuator )
pr39034.c:86:5 identifier __builtin_abort
pr39034.c:86:21 punctuator (
pr39034.c:86:22 punctuator )
pr39034.c:86:23 punctuator ;
pr39034.c:87:1 punctuator }
pr39034.c:89:1 keyword int
pr39034.c:90:1 identifier main
pr39034.c:90:6 punctuator (
pr39034.c:90:7 punctuator )
pr39034.c:91:1 punctuator {
pr39034.c:92:3 identifier runtime32
pr39034.c:92:13 punctuator (
pr39034.c:92:14 punctuator )
pr39034.c:92:15 punctuator ;
pr39034.c:93:3 identifier runtime64
pr39034.c:93:13 punctuator (
pr39034.c:93:14 punctuator )
pr39034.c:93:15 punctuator ;
pr39034.c:94:3 identifier runtime128
pr39034.c:94:14 punctuator (
pr39034.c:94:15 punctuator )
pr39034.c:94:16 punctuator ;
pr39034.c:95:3 identifier fold32
pr39034.c:95:10 punctuator (
pr39034.c:95:11 punctuator )
pr39034.c:95:12 punctuator ;
pr39034.c:96:3 identifier fold64
pr39034.c:96:10 punctuator (
pr39034.c:96:11 punctuator )
pr39034.c:96:12 punctuator ;
pr39034.c:97:3 identifier fold128
pr39034.c:97:11 punctuator (
pr39034.c:97:12 punctuator )
pr39034.c:97:13 punctuator ;
pr39034.c:99:3 keyword return
pr39034.c:99:10 constant 0
pr39034.c:99:11 punctuator ;
pr39034.c:100:1 punctuator }
